### PR TITLE
added the next Page Objects:

### PR DESCRIPTION
**Pull request by kursatiy**
ChangePasswordPage, ProfileListPage, UpdateUserPreferencesPage, UserAvatarPage, UserPreferencesOage

### CODE REVIEW

File: src/test/java/pages/ChangePasswordPage.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class ChangePasswordPage {
+
+    String fulHeaderButtonId = "//header/nav/div/div[3]/ul/li[4]/a/span/span/img";
```
Comment (archick12) [inline src/test/java/pages/ChangePasswordPage.java]: Please, remove \' and \" 
Correct: "//*[@id='change-password-submit']"
Wrong: "//*[@id=\'change-password-submit\']"
Wrong: "//*[@id=\"change-password-submit\"]"

File: src/test/java/pages/UserAvatarPage.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import java.awt.*;
+import java.awt.datatransfer.StringSelection;
+import java.awt.event.KeyEvent;
+import java.io.File;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class UserAvatarPage {
+
+
```
Comment (Artem100) [inline src/test/java/pages/UserAvatarPage.java]: Delete please empty strings in methods.

File: src/test/java/pages/ChangePasswordPage.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    String profileId = "//*[@id=\'view_profile\']";
```
Comment (Artem100) [inline src/test/java/pages/ChangePasswordPage.java]: Add please modifier **private**

File: src/test/java/pages/ProfileListPage.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package pages;
+
+import com.codeborne.selenide.Condition;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class ProfileListPage {
+
+    String userDetailsButton = "//*[@id='header-details-user-fullname']";
```
Comment (Artem100) [inline src/test/java/pages/ProfileListPage.java]: Add please modifier **private**

File: src/test/java/pages/UpdateUserPreferencesPage.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
+package pages;
+
+import com.codeborne.selenide.Condition;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class UpdateUserPreferencesPage {
+
+    String intPageSizeValue;
```
Comment (Artem100) [inline src/test/java/pages/UpdateUserPreferencesPage.java]: Add please modifier **private**

File: src/test/java/pages/UserPreferencesPage.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
+package pages;
+
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class UserPreferencesPage {
+
+    String userPreferencesEditButton = "//section/div/div[1]/div[2]/div[1]/ul/li/a/span";
```
Comment (Artem100) [inline src/test/java/pages/UserPreferencesPage.java]: Add please modifier **private**

File: src/test/java/pages/UserAvatarPage.java [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    public static void setClipboard(String str) {
+        StringSelection ss = new StringSelection(str);
+        Toolkit.getDefaultToolkit().getSystemClipboard().setContents(ss, null);
+    }
+
+    public void clickOnUserAvatar() {
+        $(By.xpath(userAvatar)).waitUntil(Condition.visible, 3000).click();
+    }
+
+    public void UploadNewAvatar() {
```
Comment (Artem100) [inline src/test/java/pages/UserAvatarPage.java]: Rewride the name metod: uploadNewAvatar()

**Pull request merged**